src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/best-practices/opentelemetry-best-practices-logs.mdx

```diff
@@ -8,52 +8,169 @@ tags:
-
-| OTLP Log Proto Field                        | New Relic `Log` field                               |
-|---------------------------------------------|-----------------------------------------------------|
-| `ResourceLogs.Resource.attributes`          | Each key/value is an attribute on the `Log` **[1]** |
-| `ScopeLogs.InstrumentationScope.name `      | `otel.library.name`                                 |
-| `ScopeLogs.InstrumentationScope.version`    | `otel.library.version`                              |
-| `ScopeLogs.InstrumentationScope.attributes` | Each key/value is an attribute on the `Log` **[1]** |
-| `LogRecord.time_unix_nanos`                 | `timestamp` **[2]**                                 |
-| `LogRecord.severity_number`                 | `severity.number`                                   |
-| `LogRecord.severity_text`                   | `severity.text`                                     |
-| `LogRecord.body`                            | `message`, and possibly parsed attributes **[3]**   |
-| `LogRecord.attributes`                      | Each key/value is an attribute on the `Log` **[1]** |
-| `LogRecord.dropped_attribute_count`         | `otel.dropped_attributes_count`                     |
-| `LogRecord.flags`                           | `w3c.flags` (integer)                               |
-| `LogRecord.trace_id`                        | `trace.id`                                          |
-| `LogRecord.span_id`                         | `span.id`                                           |
-
-**[1]**: In case of conflict in resource attributes, scope attributes, log record attributes, top level log record fields, and parsed attributes from the `LogRecord.body` **[3]**, the order of precedent (highest to lowest) is: parsed attributes from `LogRecord.body` -> top level `LogRecord.*` fields > `LogRecord.attributes` > `ScopeLogs.InstrumentationScope.attributes` > `ResourceLogs.Resource.attributes`.
-
-**[2]**: If `LogRecord.time_unix_nanos` is not present, `timestamp` is set to the time that the data was received by New Relic.
-
-**[3]**: [Log parsing](/docs/logs/ui-data/parsing/) is applied to the `LogRecord.body` to attempt to extract attributes from plain log text. For example, if a JSON structured log format is used, the key / values become attributes on the resulting log. This is particularly useful when collecting logs from files or stdout. In this case, it's common to have no resource attributes associated with the log record (required for [APM service correlation](#service-correlation)) and no value for `LogRecord.trace_id` / `LogRecord.span_id` (required for [trace correlation](#trace-correlation)). Correlation will function as intended if the required fields can be successfully parsed.
+New Relic supports two primary workflows for ingesting logs generated with OpenTelemetry:
```

Comment:
This phrasing isn't correct. These aren't new relic's workflows and we don't explicitly support them. These are the workflows identified / documented by the OpenTelemetry project. At the end of the day, New Relic supports OTLP ingest.

Reply:
@jack-berg I'm open to changing the phrasing, but I want to keep it New Relic-focused and communicate that we "support" the two OTLP ingest workflows.

How about something like "To send OTLP-ingested logs to New Relic, use one of the following OpenTelemetry workflows:"

I'm open to suggestions.